Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG PARENT_VERSION=2.2.1-node20.9.0
-ARG PORT=3000
+ARG PARENT_VERSION=2.1.2-node18.11.0
```

Comment:
Any particular reason we want to downgrade the node version?

Reply:
For now, only to keep it in line with what exists in ffc-pay. As John set this up using the template I have not upgraded the node version on our existing version in order to check that it all still works.